etc/adminhtml/system.xml

```diff
@@ -239,6 +239,14 @@
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                 </field>
             </group>
+            <group id="nosto_account_info" translate="label" type="text" sortOrder="60" showInDefault="0"
+                   showInWebsite="0" showInStore="1">
+                <label>Account Details and Tokens</label>
+                <field id="nosto_account_info_fields" translate="label comment" type="text" sortOrder="20"
+                       showInDefault="1" showInWebsite="1" showInStore="1">
```

Comment:
Why would we show in website and default when tokens are only persisted on a store-level?

Reply:
They are momentally dependent on the parent group configuration.